app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -387,11 +403,13 @@ export const mapStateToProps = (state, ownProps) => {
   const references = ownProps.references
     ? viewerModule.selectors.parseReferences(ownProps.doc, ownProps.references)
     : relevantReferences;
+  const defaultLanguage = state.locale;
```

Comment:
This naming is confusing.   This is not the "defaultLanguage", this is the "currentLanguage".